src/control/run_go_tests.sh

```diff
@@ -44,10 +44,16 @@ function setup_environment()
 
 	source "${build_source}"
 
-	LD_LIBRARY_PATH="${SL_PREFIX}/lib:${SL_SPDK_PREFIX}/lib:${LD_LIBRARY_PATH}"
-	export LD_LIBRARY_PATH
-	export CGO_LDFLAGS="-L${SL_SPDK_PREFIX}/lib -L${SL_PREFIX}/lib"
-	export CGO_CFLAGS="-I${SL_SPDK_PREFIX}/include"
+	# sigh, appease the linter...
+	LD_LIBRARY_PATH="${SL_PREFIX}/lib"\
+":${SL_SPDK_PREFIX}/lib"\
+":${SL_HWLOC_PREFIX}/lib"
+	CGO_LDFLAGS="-L${SL_SPDK_PREFIX}/lib"\
```

Comment:
(lint) The double quotes around this do nothing. Remove or escape them. [SC2140]